src/go/rpk/pkg/vcloud/yak/client.go

```diff
@@ -45,33 +45,53 @@ func (yc *YakClient) GetNamespaces() ([]*Namespace, error) {
 		return nil, ErrLoginTokenMissing{err}
```

Comment:
This commit could be squashed onto the 1st one, right?

Reply:
mm maybe, can we keep it like this thought? 🙏